ext/opentelemetry-ext-jaeger/setup.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+import setuptools
+from setuptools.command.install import install
+
+BASE_DIR = os.path.dirname(__file__)
+VERSION_FILENAME = os.path.join(
+    BASE_DIR, "src", "opentelemetry", "ext", "jaeger", "version.py"
+)
+PACKAGE_INFO = {}
+with open(VERSION_FILENAME) as f:
+    exec(f.read(), PACKAGE_INFO)
+
+BASE_CMD = """docker run --user `id -u` -v "$PWD:/data" \
```

Comment:
This adds an install-time dependency on docker? That is a pretty big dependency.

Also it looks like the install depends on the pwd being the dir that includes `setup.py`?

```
% python ext/opentelemetry-ext-jaeger/setup.py install
running install
[FAILURE:arguments:1] Could not open input file with realpath: /data/thrift/agent.thrift
[FAILURE:arguments:1] Could not open input file with realpath: /data/thrift/zipkincore.thrift
[FAILURE:arguments:1] Could not open input file with realpath: /data/thrift/jaeger.thriftrunning build
```

I understand not wanting to check in the generated thrift files, but we should consider generating them before releases instead. As it is now, a user would have to have `docker` installed to install this package via `setup.py install`. What about if they're installing from a wheel? How do we include the generated files in the distribution?